docs/content/preview/troubleshoot/active-session-history.md

```diff
@@ -0,0 +1,350 @@
+| wait_event_component | text | There are three components: YSQL, YCQL, and YB-TServer. |
+| wait_event_class | text | Every wait event will have a class associated with it. |
+| wait_event | text | Provides insight into what the RPC is waiting on. |
+| wait_event_type | text | Type of the wait event such as CPU, WaitOnCondition, Network, Disk IO, and so on. |
+| wait_event_aux | text | Additional information for the wait event. For example, tablet ID for yb-tserver wait events. |
+| top_level_node_id | UUID | 16-byte yb-tserver UUID of the YSQL/YCQL node where the query is being executed. |
+| query_id | bigint | Query ID as seen on the `/statements` endpoint. This can be used to join with [pg_stat_statements](../../explore/query-1-performance/pg-stat-statements/)/ycql_stat_statements. For background activities, query ID will be a known constant (for example, log appender is 1, flush is 2, compaction is 3, consensus is 4). |
+| ysql_session_id | bigint | Same as `PgClientSessionId`. This is 0 for YCQL and background activities, as this is a YSQL specific field. |
+| client_node_ip | text | Client IP for the RPC. For YSQL, it will be the client node from where the query is generated. For YB-TServer, it will be the YSQL/TServer node from where the RPC originated. |
+| sample_weight | float | If in any sampling interval there are too many events, YugabyteDB only collects `yb_ash_sample_size` samples/events. Based on how many were sampled, weights are assigned to the collected events. <br><br>For example, if there are 200 events, but only 100 events were collected, each of the collected sample will have a weight of (200 / 100) = 2.0 |
+
+### yb_local_tablets
+
+This view provides tablet ID metadata, state, and role information. This information is also available on the `<yb-tserver_ip>:9000/tablets` endpoint.
+
+| Column | Type | Description |
+| :----- | :--- | :---------- |
+| tablet_id | text | 16 byte UUID of the tablet. Same as `/tablets` endpoint. |
+| table_id | text | 16 byte UUID of the table which the tablet is part of. |
+| table_type | text | Type of the table from YSQL, YCQL, System, or Unknown. |
+| namespace_name | text | Name of the database or the keyspace. |
+| ysql_schema_name | text | YSQL schema name. Empty for YCQL, System, and Unknown table types. |
+| table_name | text | Name of the table which the tablet is part of. |
+| partition_key_start| bytea | Start key of the partition (inclusive). |
+| partition_key_end  | bytea | End key of the partition (exclusive).|
+
+### ycql_stat_statements
+
+This view provides YCQL statement metrics that can be joined with YCQL wait events in [yb_active_session_history](#yb_active_session_history) table.
+
+| Column | Type | Description |
+| :----- | :--- | :---------- |
+| query_id | int8 | Hash code to identify identical normalized queries. |
```

Comment:
This one should be `queryid` instead of `query_id`. I agree this is confusing because the one in `yb_active_session_history` is `query_id`.